analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -693,7 +693,7 @@ class Pip(
         }
 
         val declaredLicenses = sortedSetOf<String>()
-        map["License"]?.let { declaredLicenses += it.single() }
+        getLicenseFromLicenseField(map["License"]?.single())?.let { declaredLicenses += it }
```

Comment:
Commit message nit: s/by/By/

Also if you agree to rename the function, remember to also rename it in this commit message.

Reply:
Updated the commit message.